server/src/main/java/io/deephaven/server/session/SessionService.java

```diff
@@ -333,21 +338,41 @@ public SessionState getSessionForAuthToken(final String token) throws Authentica
                 if (session != null) {
                     return session;
                 }
-            } catch (IllegalArgumentException ignored) {
+            } catch (IllegalArgumentException | InvalidUuidException ignored) {
```

Comment:
Do we expect IllegalArgumentException to ever get hit?

Reply:
Not that I'm aware of, but wanted to broaden what we expected, continue to ignore things we've previously ignored. If a simple runtime error happens in trying to obtain the session, we can _probably_ ignore that call?

Open to your feedback on how this should be handled, I went with the broader interpretation.

Reply:
I'm generally a fan of only catching the exceptions you can reasonably _expect_. That said, happy to only widen as part of this PR.

Reply:
The IAE was intended for the UUID parsing - which I didn't realize was incorrect when swapping from UUID to this UuidCreator impl.